cmd/sessionstorage.go

```diff
@@ -0,0 +1,238 @@
+//
+// Copyright (c) 2023 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package cmd
+
+import (
+	authApi "github.com/SSHcom/privx-sdk-go/api/auth"
+	"github.com/spf13/cobra"
+)
+
+type sessionStorageOptions struct {
+	userID    string
+	sourceID  string
+	sessionID string
+	sortkey   string
+	sortdir   string
+	offset    int
+	limit     int
+}
+
+func init() {
+	rootCmd.AddCommand(sessionStorageCmd())
+}
+
+//
+//
+func sessionStorageCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sessions",
+		Short: "List and manage sessions",
+		Long:  `List and manage sessions`,
+		Example: `
+	privx-cli sessions [access flags]
+		`,
+		SilenceUsage: true,
+	}
+
+	cmd.AddCommand(userSessionsShowCmd())
+	cmd.AddCommand(sourceSessionsShowCmd())
+	cmd.AddCommand(sessionsSearchCmd())
+	cmd.AddCommand(terminateSessionCmd())
+	cmd.AddCommand(terminateUserSessionsCmd())
+
+	return cmd
+}
+
+//
+//
+func userSessionsShowCmd() *cobra.Command {
+	options := sessionStorageOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "show-by-user",
+		Short: "Get sessions by userID",
+		Long:  `Get sessions by userID. Fetch valid sessions for specified user`,
+		Example: `
+	privx-cli sessions show-by-user [access flags] --id <USER-ID>
+		`,
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return userSessionsShow(options)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&options.userID, "id", "", "User ID")
+	flags.IntVar(&options.offset, "offset", 1, "where to start fetching the items")
```

Comment:
Is there a reason to set the offset to 1 or is it just a typo?

Reply:
typo fixed